src/adaptors/moola/index.js

```diff
@@ -0,0 +1,59 @@
+const utils = require('../utils');
+
+
+const poolsFunction = async () => {
+    const apyData = await utils.getData(
+        'https://v2-srv-data-frm-smrt-cntract.herokuapp.com/get/getReserveData'
+    );
+
+    //Pools return stable APR
+
+    const celoPool = {
+        pool: '0x7D00cd74FF385c955EA3d79e47BF06bD7386387D',
+        chain: utils.formatChain('celo'),
+        project: 'moola',
+        symbol: utils.formatSymbol('Celo'),
+        tvlUsd: Number(apyData.data[0].totalDeposited),
+        apy: apyData.data[0].stableBorrowAPR,
```

Comment:
on defillama/yields we do not display borrow rates, but lending rates (in your case, this would be the `apy` field i guess

Reply:
👍 i always get confused >_<